From 22350aace8e06a1fbe37bf84ddbff8d882c14283 Mon Sep 17 00:00:00 2001
From: Murod Komilov <murodkomilov18@gmail.com>
Date: Tue, 28 Nov 2023 10:24:56 +0500
Subject: [PATCH] Changed two lines in firstFile.txt from git_1

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index fbb00c1..db9d90e 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -1,5 +1,5 @@
-Line 1
-Line 2
+Line 1-modified
+Line 2-modified
 Line 3
 Line 4
 Line 5
-- 
2.34.1

